### PR TITLE
Us0028 role filtering

### PR DESCRIPTION
**Pull request by CallumE74**

### CODE REVIEW

**Review by Adaray786 — APPROVED** [review]
lgtm lgtm

**Review by sammyperry101 — APPROVED** [review]

File: src/main/java/org/kainos/ea/api/JobBandService.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import org.kainos.ea.cli.JobBand;
+import org.kainos.ea.client.FailedToGetJobBandException;
+import org.kainos.ea.client.JobBandNotFoundException;
+import org.kainos.ea.db.JobBandDao;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class JobBandService {
```
Comment (TomBillingtonUK) [inline src/main/java/org/kainos/ea/api/JobBandService.java:11]: Missing unit tests

File: src/main/java/org/kainos/ea/db/JobBandDao.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import org.kainos.ea.cli.JobBand;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JobBandDao {
```
Comment (TomBillingtonUK) [inline src/main/java/org/kainos/ea/db/JobBandDao.java:12]: Missing unit tests

File: src/main/java/org/kainos/ea/resources/JobBandController.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import org.kainos.ea.client.FailedToGetJobBandException;
+import org.kainos.ea.client.JobBandNotFoundException;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Api("Team 2 Band API")
```
Comment (TomBillingtonUK) [inline src/main/java/org/kainos/ea/resources/JobBandController.java:14]: Missing unit and intergration tess

**Pull request merged**